doc/api/fs.md

```diff
@@ -760,12 +760,46 @@ no effect on Windows (will behave like `fs.constants.F_OK`).
 
 The final argument, `callback`, is a callback function that is invoked with
 a possible error argument. If any of the accessibility checks fail, the error
-argument will be an `Error` object. The following example checks if the file
-`/etc/passwd` can be read and written by the current process.
+argument will be an `Error` object. The following examples check if
+`package.json` exists, and if it is readable or writable.
 
 ```js
-fs.access('/etc/passwd', fs.constants.R_OK | fs.constants.W_OK, (err) => {
-  console.log(err ? 'no access!' : 'can read/write');
+// Check if `package.json` exists in the current directory.
+fs.access('./package.json', fs.constants.F_OK, (err) => {
+  console.log(err ? 'package.json does not exist' : 'package.json exists');
+});
+
+// Check if `package.json` file is readable.
+fs.access('./package.json', fs.constants.R_OK, (err) => {
+  console
+    .log(err ? 'package.json is not readable' : 'package.json is readable');
+});
+
+// Check if `package.json` is writable.
+fs.access('./package.json', fs.constants.W_OK, (err) => {
+  console.log(
+    err ?
```

Comment:
It seems this can be unwrapped now like in the block above, to spare some lines)